cmd/stack.go

```diff
@@ -176,6 +177,44 @@ nitric stack down -s aws -y`,
 	Args: cobra.ExactArgs(0),
 }
 
+var stackPreviewCommand = &cobra.Command{
+	Use:     "preview [-s stack]",
+	Short:   "Preview the deployment of a stack",
+	Long:    `Preview the deployment of a stack`,
+	Example: `nitric stack down -s aws`,
+	Run: func(cmd *cobra.Command, args []string) {
+		s, err := stack.ConfigFromOptions()
+
+		if err != nil && strings.Contains(err.Error(), "No nitric stacks found") {
```

Comment:
This block doesn't work for me. Yes sends the error "No nitric stacks found".
Same thing happens to the up command. Could be an unrelated issue to this PR

Reply:
This was removed from `up` as well when it was refactored, there are a few lines commented out. Will update now.